code/game/objects/structures/watercloset.dm

```diff
@@ -529,7 +529,7 @@
 			if(B.cell.charge > 0 && B.status == 1)
 				flick("baton_active", src)
 				var/stunforce = B.stamforce
-				user.Knockdown(stunforce * 2)
+				user.DefaultCombatKnockdown(stunforce)
```

Comment:
i'm going to assume that this is an undocumented nerf since all other Knockdown to DefaultCombatKnockdown changes up til here have been the same value

Reply:
woops